packages/components/src/components/omnibox/actions/Filter.ts

```diff
@@ -412,12 +414,18 @@ export class FilterAction implements Action {
 
         return strValues.map(strValue => {
             const value = `"${col.shortCaption}" ${operator} ${strValue}`;
+            let displayValue = value;
```

Comment:
Looks like we don't need the `value` variable here anymore:
```suggestion
            let displayValue = `"${col.shortCaption}" ${operator} ${strValue}`;
```

Reply:
it's actually still needed, see the return object contains value

> Looks like we don't need the `value` variable here anymore: